### ISSUE TITLE
[BUG] Spiel friert ein/ Game freezes - Error:  PathFinderModule.lua:1958: bad argument #2 to 'max'

### ISSUE THREAD

**Issue by chroboss** [Stephan-S/FS19_AutoDrive#2027]
AutoDrive Version: 1.1.1.4
Release or selbsterstelltes zip: Release
Karte (falls relevant): Cornbelt oP 1.3 (Farmer Andy)
Fahrzeug(art) (falls relevant):  kein spezifisches
[log-CB.txt](https://github.com/Stephan-S/FS19_AutoDrive/files/6847392/log-CB.txt)

**Beschreibe den Bug**
Spiel friert plötzlich ein und ist nur mit Task Manager zu beenden

**Comment by Iwan1803** [comment]
Wann genau in welcher Situation? Ist der Fehler reproduzierbar?

**Comment by chroboss** [comment]
Nein, den Fehler kann ich nicht reproduzieren, da er spontan auftritt. Jedoch jedes mal beim durchtabben der Fahrzeuge, jeweils in ein aktives Farzeug, das aktuell mit Feldarbeit (CP),  oder Transporten(AD) beschäftigt ist. Bei inaktiven Fahzeugen ist das bislang nicht aufgetreten. Info: Gleichzeitig sind weitere AD und CP Helfer aktiv.

**Comment by fsplayer4** [comment]
bei  mir friert es gleich ein, liegt also nicht an einenm Fahrzeug

**Comment by Foss-ops** [comment]
Bei mir auch,  ist nicht reproduzierbar, kann stundenlang gutgehen, oder auch  nicht.

Ebenfalls 1.1.1.4 Release. Selbe Error Message im Log,

**Comment by fsplayer4** [comment]
Habe leider immer noch das Problem
